src/test/java/org/kpmp/atlasMessage/AtlasMessageTest.java

```diff
@@ -0,0 +1,46 @@
+package org.kpmp.atlasMessage;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+
+public class AtlasMessageTest {
+    @Mock
+    private AtlasMessage atlasMessage;
+    @BeforeEach
+    public void setUp() throws Exception {
+        atlasMessage = new AtlasMessage();
+    }
+
+    @AfterEach
+    public void tearDown() throws Exception {
+        atlasMessage = null;
+    }
+
+    @Test
+    public void testGetAtlasMessage() throws Exception {
```

Comment:
These should be in separate tests...one for each method. Otherwise when you get a failure, you only know that one thing is broken until you fix that, then you will see the next broken thing, etc.

It also makes the code cleaner/easier to read.